spel/userdata/tmpfsroot-el7.cloud

```diff
@@ -12,73 +21,96 @@ system_info:
     lock_passwd: true
     sudo: ["ALL=(root) NOPASSWD:ALL"]
 
+bootcmd:
+  - setenforce 0
```

Comment:
We can give this a try, but there was definitely a reason for ignoring failures. Might need to make this more robust, checking for selinux pkgs or configs or something, to determine when to run this and when to skip it.

Reply:
Previous invocation was in `runcmd` where, prior to calling the `setenforce` you'd invoked `set -e` which would have cause the entire `runcmd` block to exit if `setenforce` failed. In the context of the above `bootcmd`, the `setenforce` is able to fail without causing an exit.

Reply:
Ahh, good point. Might be nice to capture any failure message with the new console logging though. I don't think we'll get that by keeping it in the cloud-config section?

Reply:
Yeah. Easy enough to move it back. Part of why I wanted to do the mixed-MIME was to make it easy to add more-useful `&& <ACTION> || <ACTION` content. With the opening logger-block in the script, anything that goes to STDERR/STDOUT becomes available on the instance-console rather than just the cloud-init log that goes poof. 

I'd consisdered adding `--verbose` to the `cp` actions, but that would have resulted in ridonkulous amounts of stuff going to the console and logfiles.

Reply:
That's fine. It really is a shell script, so it makes more sense this way. I was honestly just being lazy before, when adding the cloud-config to set the `spel` suer, and didn't want to lookup the mixed-mime setup. Was easier to just tab it over and use runcmd. 😁